### PR TITLE
Add System Block Diagram

### PR DESCRIPTION
**Pull request by trau3**

### CODE REVIEW

**Comment by kroosvi** [comment]
Code Space was used to display the website, everything looks pretty good. Works perfectly, but is this PR for updating the pictures on the website? If so, I don't see any pictures on the website.

**Review by kroosvi — COMMENTED** [review]
The first chart would not be easy to follow if I had no idea what this project was going on,  to see the sequence of events happening is on the harder side.  The chart is pretty good if one has some idea of how this application should behave.

**Comment by trau3** [comment]
Which part is hard to follow in the sequence of events?




On Saturday, September 16, 2023, 12:10 AM, Kroos Xiang ***@***.***> wrote:




@kroosvi commented on this pull request.

This chart would not be easy to follow if I had no idea what this project was going on, to see the sequence of events happening is on the harder side. The chart is pretty good if one has some idea of how this application should behave.

**Review by kroosvi — APPROVED** [review]
I was on the nitpicking side the first time around, I think it does convey the whole idea pretty well, I was just having trouble following it along with the perspective of a person who just saw this project the first time.

**Comment by kroosvi** [comment]
The chart showed up in Codespace pretty well, nothing was distorted or unclear.
